recipe/meta.yaml

```diff
@@ -181,6 +182,64 @@ outputs:
         Compiler for CUDA applications.
       doc_url: https://docs.nvidia.com/cuda/index.html
 
+  - name: cuda-crt-tools
+    files:       # [linux]
+      - bin/crt  # [linux]
```

Comment:
Should we mirror the same structure on Linux & Windows? It might make maintenance and usage simpler

```suggestion
    files:
      - bin/crt          # [linux]
      - Library\bin\crt  # [win]
```